torchdynamo/config.py

```diff
@@ -93,3 +93,5 @@
 # Not all backends support scalars. Some calls on torch.Tensor (like .item()) return a scalar type.
 # When this flag is set to False, we introduce a graph break instead of capturing.
 capture_scalar_outputs = False
+
+export_guards = True
```

Comment:
This config controls two things:
1) Prints
2) If `guard_export_fn!=None` will trigger an assert error.

These seem like seperate things.  For 1, we should either tie it to debug/trace or give it a print related name.

For 2, could we just look at `guard_export_fn is None` to decide what to do?  I'm not sure why we need both a config and a kwarg.